src/test/sectionSpec/fraSpecs.ts

```diff
@@ -2471,7 +2470,7 @@ export const FraSpecs: Record<string, SectionSpec> = {
                   calcFormula: '(growingStockAvg.plantedForest * forestCharacteristics.plantedForest) / 1000',
                   readonly: false,
                   validateFns: [
-                    `validatorSubCategory(growingStockTotal.forest,[growingStockTotal.naturallyRegeneratingForest,growingStockTotal.plantedForest])`,
+                    `validatorSubCategory(growingStockTotal.plantedForest,[growingStockTotal.plantationForest,growingStockTotal.otherPlantedForest])`,
```

Comment:
the formula should not be changed. the rule is plantedForest+naturallyRegeneratingForest <= forest